.circleci/verbatim-sources/workflows-nightly-android-binary-builds.yml

```diff
@@ -4,36 +4,24 @@
           requires:
             - setup
           docker_image: "308535385114.dkr.ecr.us-east-1.amazonaws.com/pytorch/pytorch-linux-xenial-py3-clang5-android-ndk-r19c:347"
-          filters:
-            branches:
-              only: nightly
```

Comment:
removing these filters effectively runs binary jobs on the current target PR branch (for this PR it's v1.3.0), and that's pretty much what we want